FileManager/Delegates/FilesManagerServiceDelegate.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  FilesManagerServiceDelegate.swift
+//  FileManager
+//
+//  Created by Екатерина Неделько on 5.06.22.
+//
+
+import Foundation
+import UIKit
+
+protocol FilesManagerServiceDelegate {
+    var filesTableView: UITableView { get set }
+    var filesCollectionView: UICollectionView { get set }
```

Comment:
`FilesManagerService` shouldn't know anything about the UI because it's a model so it shouldn't have these properties